editor/plugins/sprite_2d_editor_plugin.cpp

```diff
@@ -449,6 +445,27 @@ void Sprite2DEditor::_add_as_sibling_or_child(Node *p_own_node, Node *p_new_node
 	p_new_node->set_owner(get_tree()->get_edited_scene_root());
 }
 
+void Sprite2DEditor::_sync_sprite_resize_mode() {
+	if (node != nullptr) {
+		node->_editor_set_dragging_to_resize_rect(resize_region_rect->is_pressed());
+	}
+}
+
+void Sprite2DEditor::_update_sprite_resize_mode_button() {
+	if (node == nullptr) {
+		return;
+	}
+	resize_region_rect->set_disabled(!node->is_region_enabled());
+	resize_region_rect->set_pressed(node->_editor_is_dragging_to_resiz_rect());
+}
+
+void Sprite2DEditor::_update_sprite_resize_mode_tooltip() {
+	if (node != nullptr && !node->is_region_enabled()) {
+		resize_region_rect->set_tooltip_text("\n" + TTR("Sprite's region needs to be enabled in the inspector."));
```

Comment:
```suggestion
		resize_region_rect->set_tooltip_text(TTRC("Sprite's region needs to be enabled in the inspector."));
```
This seems to work correctly.